packages/flutter/lib/src/services/binary_messenger.dart

```diff
@@ -0,0 +1,153 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:typed_data';
+import 'dart:ui' as ui;
+
+import 'package:flutter/foundation.dart';
+
+typedef _MessageHandler = Future<ByteData> Function(ByteData message);
```

Comment:
this could be public, since you use the same signature in public APIs.

Reply:
Done.